recipes/libevent/build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Needed to ensure our OpenSSL and
+# not the system one is used on OS X.
+export LIBRARY_PATH="${PREFIX}/lib"
+
+export CFLAGS="-I${PREFIX}/include ${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include ${CPPFLAGS}"
+export LDFLAGS="-L${PREFIX}/lib ${LDFLAGS}"
+
+# Set the fallback library environment variable.
+if [[ `uname` == 'Darwin' ]];
+then
+    export LIBRARY_SEARCH_VAR=DYLD_FALLBACK_LIBRARY_PATH
+else
+    export LIBRARY_SEARCH_VAR=LD_LIBRARY_PATH
+fi
+
+
+chmod +x ./autogen.sh
+
+./autogen.sh
+./configure --prefix="${PREFIX}"
+make
+eval ${LIBRARY_SEARCH_VAR}="${PREFIX}/lib" make check
```

Comment:
This test section is causing some sort of hanging behavior so it may need to be commented and skipped for now.